packages/markup-frontstage/package.json

```diff
@@ -0,0 +1,114 @@
+{
+  "name": "@bentley/markup-frontstage-react",
+  "version": "1.0.0",
+  "description": "Markup Frontstage React",
+  "license": "MIT",
+  "main": "lib/markup-frontstage-react.js",
+  "typings": "lib/markup-frontstage-react",
+  "scripts": {
+    "copy:assets": "cpx \"./src/**/*.scss\" ./lib && cpx \"./public/**/*\" ./lib/public",
+    "pseudolocalize": "betools pseudolocalize --englishDir ./public/locales/en --out ./lib/public/locales/en-PSEUDO",
+    "build": " tsc 1>&2 && npm run copy:assets && npm run pseudolocalize",
+    "build-prod": " tsc -p tsconfig.json 1>&2 && npm run copy:assets",
```

Comment:
This script will never be run, can probably be removed.

Reply:
removed please :)